components/camel-micrometer/pom.xml

```diff
@@ -108,6 +108,13 @@
             <version>${assertj-version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.awaitility</groupId>
+            <artifactId>awaitility</artifactId>
+            <version>4.2.0</version>
```

Comment:
This should come as property placeholder from Parent POM

Reply:
thanks, fixed